src/components/WeeklyShow.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from "prop-types";
+import styles from "../styles/WeeklyShow.module.css";
+
+export default function WeeklyShow({ show }){
+
+  if (show === {} || show === undefined){
+    return <div className={styles.WeeklyShow} />
+  }
+
+  const colors = {
```

Comment:
perhaps we should move this to `component-utils.js`?

Reply:
hmmm, that file came about after this branch was made. Maybe do that after this is merged?

Reply:
Yeah, that makes sense